src/blip.sol

```diff
@@ -72,7 +72,7 @@ contract Blipper is Clipper {
         uint256 ask = rmul(tab, WAD) / rdiv(mid, bee);
 
         // how much dai to get for the entire collateral
-        uint256 bid = mul(wmul(lot, rmul(mid, bee)), RAY);
+        uint256 bid = rdiv(mul(wmul(lot, mid), RAY), bee);
```

Comment:
This will be very prone to overflow because the `rdiv` is internally doing a multiplication of a 45 decimal digit precision number with `RAY` (upper bound on `lot*mid` in floating-point terms will be about 10^5). I suggest structuring the calculation instead as:

`mul(mul(lot, 10**9), rdiv(mid, bee)))`

You can also eliminate the `bid` variable and make that expression the RHS of the current line 83.